fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/iterators/RdfStream.java

```diff
@@ -57,7 +57,7 @@
 
     /**
      * Constructor that begins the stream with proffered triples.
-     * 
```

Comment:
Mixing non-functional format changes with substantive commits makes code review more painful and code archaeology more difficult.